### PR TITLE
chore: [SIW-1015] Added Azure function programming model v4

### PR DESCRIPTION
**Pull request by silvicir**
#### List of Changes

Added Azure function programming model v4 and replaced `wallet instance attestation` with `wallet instance`

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by grausof — APPROVED** [review]
Great work :rocket:

File: apps/io-func-wallet-solution-backend/src/infra/http/handlers/create-wallet-attestation.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+export const CreateWalletAttestationHandler = H.of((req: H.HttpRequest) =>
+  pipe(
+    req,
+    requireWalletAttestationRequest,
+    RTE.chain(({ walletAttestationRequest }) =>
+      createWalletAttestation(walletAttestationRequest)
+    ),
+    RTE.map(createdEntityStatementJwt),
+    RTE.orElseW(logErrorAndReturnResponse)
+  )
```
Comment (gunzip) [inline apps/io-func-wallet-solution-backend/src/infra/http/handlers/create-wallet-attestation.ts:47]: you can use flow

Comment (gunzip) [reply apps/io-func-wallet-solution-backend/src/infra/http/handlers/create-wallet-attestation.ts:47]:
```suggestion
export const CreateWalletAttestationHandler = H.of(flow(
    requireWalletAttestationRequest,
    RTE.chain(({ walletAttestationRequest }) =>
      createWalletAttestation(walletAttestationRequest)
    ),
    RTE.map(createdEntityStatementJwt),
    RTE.orElseW(logErrorAndReturnResponse)
  )
```

Comment (silvicir) [reply apps/io-func-wallet-solution-backend/src/infra/http/handlers/create-wallet-attestation.ts:47]: I have not touched the code, just renamed the file. We will do in another PR

**Pull request merged**